src/solc_0.5/Land/erc721/LandBaseTokenV3.sol

```diff
@@ -426,6 +471,13 @@ contract LandBaseTokenV3 is ERC721BaseTokenV2 {
         }
     }
 
+    /// @param land land to be cleared
+    /// @param quadMinted array of lands minted
+    /// @param index array size
+    /// @param landMinted number of lands transferred
+    /// @param quadCompareSize size to compare with
+    /// @return
+    /// @return
```

Comment:
missing description

Reply:
i removed the double return, can you help on the comment, i can't figure exactly what it returns

Reply:
can you help on the comment, i can't figure exactly what it returns